File: TEO/Mav.Models/ViewModels/CertificateViewModel.cs
using Mav.Models.SearchModels;
using System;
using System.Collections.Generic;

namespace Mav.Models.ViewModels
{
    public class CertificateViewModel
    {
        public int ID { get; set; }

        public Guid KeyID { get; set; }

        public string UserID { get; set; }

        public string UserName { get; set; }

        public string CentreName { get; set; }

        public string AssessmentName { get; set; }

        public int YearID { get; set; }

        public string YearDisplay { get; set; }

        public int TotalScore { get; set; }

        public int MaxScore { get; set; }

        public DateTime? SubmittedOn { get; set; }

        public string Grade { get; set; }

        public IList<UserAssessmentSectionListingModel> Sections { get; set; }
    }
}

File: TEO/Mav.UI/Mav.Ui/src/app/models/certificate.view.model.ts
import { UserAssessmentSectionListingModel } from './search/user.assessment.section.listing.model';

export class CertificateViewModel {

    public id: number;

    public keyID: string;

    public userID: string;

    public userName: string;

    public centreName: string;

    public assessmentName: string;

    public yearID: number;

    public yearDisplay: string;

    public totalScore: number;

    public maxScore: number;

    public submittedOn: Date;

    public grade?: string;

    public sections: UserAssessmentSectionListingModel[];

}
